python/xraylib_np.pyx

```diff
@@ -1491,7 +1492,7 @@ Q1_SHELL = xrl.Q1_SHELL
 Q2_SHELL = xrl.Q2_SHELL
 Q3_SHELL = xrl.Q3_SHELL
 
-def AtomicWeight(cnp.ndarray[long, ndim=1] Z not None):
+def AtomicWeight(cnp.ndarray[int64_t, ndim=1] Z not None):
```

Comment:
@t20100 I had to make a change to your patch as it didn't work properly on Windows with the numpy example:

```
ValueError: Buffer dtype mismatch, expected 'long' but got 'long long'
```

This is due to Windows treating `long`s as 32-bit integers instead of 64-bit.